lib/PlacesProvider.js

```diff
@@ -719,6 +721,22 @@ Links.prototype = {
   }),
 
   /**
+   * Get all history items
```

Comment:
Can you add a comment about what this is for? i.e. to determine frequency of visits